src/app/f2f/controllers/results.js

```diff
@@ -181,6 +183,31 @@ class PostcodeSearchController extends BaseController {
       req.sessionModel.set("payLoadValues", locals.payLoadValues);
 
       callback(err, locals);
+    } catch (error) {
+        const headers = {
+          "x-govuk-signin-session-id": req.session.tokenId,
+        };
```

Comment:
This catch all looks duplicated from the abort controller, could we extract it out into a util function?

Reply:
Also apologies if I've missed something but I can't see anything in the ticket about aborting the users session for this failure?

Reply:
So in AC1 I interpreted the routing to IPV as an abort- do you think it would be better to make a new endpoint for this particular scenario? And yes will extract to util  👍🏻